Linting requires eslint-plugin-jsx-a11y, not in package.json

Forked the repo. Ran:

```
npm install
```
then:
```
npm run test
```
Got an error that boils down to:
```
ESLint couldn't find the plugin "eslint-plugin-jsx-a11y".
```

My guess is that you have it installed globally on your machine, as I didn't see it in the package.json. Thought this might be something you wanted to add as a devDependency.

In an effort to provide more detail, I have installed local to the project eslint-plugin-jsx-a11y. After doing so, it errored asking for the eslint-plugin-react package. I installed that next. I then ran `npm run test` once more and received the following error:

```
Referenced from: airbnb
Referenced from: /Library/WebServer/Documents/Work/redux-pack/.eslintrc
Error: /Library/WebServer/Documents/Work/redux-pack/node_modules/eslint-config-airbnb/rules/react-a11y.js:
        Configuration for rule "jsx-a11y/anchor-has-content" is invalid:
        Value "" is the wrong type.
```

In addition, a fresh `npm install` with this package.json returns a:
```
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^2.2.2, required by eslint-config-airbnb@12.0.0
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -37,30 +37,6 @@
     "no-console": 0,
     // Disable `arrow-parens` because it conflicts with Flow.
     "arrow-parens": 0,
-    // Do not enforce component method order
-    "no-restricted-properties": [2, {
-      "object": "arguments",
-      "property": "callee",
-      "message": "arguments.callee is deprecated,"
-    }, {
-      "property": "__defineGetter__",
-      "message": "Please use Object.defineProperty instead.",
-    }, {
-      "property": "__defineSetter__",
-      "message": "Please use Object.defineProperty instead.",
-    }, {
-      "object": "NativeModules",
-      "message": "Use require('react-native-safe-module').module(...) instead."
-    }, {
-      "object": "requireNativeComponent",
-      "message": "Use require('react-native-safe-module').component(...) instead."
-    }, {
-      "property": "NativeModules",
-      "message": "Use require('react-native-safe-module').module(...) instead."
-    }, {
-      "property": "requireNativeComponent",
-      "message": "Use require('react-native-safe-module').component(...) instead."
-    }],
     "prefer-object-spread/prefer-object-spread": 2
   },
   "globals": {

--- a/package.json
+++ b/package.json
@@ -43,9 +43,10 @@
     "babel-preset-react-native": "^1.9.0",
     "eslint": "^3.11.1",
     "eslint-config-airbnb": "^12.0.0",
-    "eslint-plugin-flowtype": "^2.3.0",
     "eslint-plugin-import": "^1.16.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-prefer-object-spread": "^1.0.0",
+    "eslint-plugin-react": "^6.8.0",
     "normalizr": "^2.3.0",
     "redux": "^3.6.0"
   }

